### ISSUE TITLE
Incompatible with 5.61+?

### ISSUE THREAD

**Issue by jofranz** [veda-consulting-company/uk.co.vedaconsulting.gdpr#335]
```
➜  uk.co.vedaconsulting.gdpr (master) check-extensions-for-removed-core-code.sh -d .                                                                                                                             ✭
./CRM/Gdpr/Form/Search/GroupcontactDetails.php:
CRM_Core_Form_Date::buildDateRange($form, 'subscription_date', 1, '_low', '_high', E::ts('From'), FALSE);

```

@eileenmcnaughton 
Touches #330

**Comment by eileenmcnaughton** [comment]
Yeah -  looks like that deprecation was in 2020 so it would definitely be good to stop calling if from this extension - it seems this extension calls a few core furnction that are not supported for non-core use

**Comment by eileenmcnaughton** [comment]
Note I don't think the presence of deprecation warnings makes it incompatible - live sites should not be displaying deprecation warnings (but staging & dev should so they they do get fixed)

**Comment by jofranz** [comment]
@eileenmcnaughton Correct me if I'm wrong but `CRM_Core_Form_Date::buildDateRange` has been kicked out of CiviCRM core which breaks this extension if this method is called.

**Comment by eileenmcnaughton** [comment]
@jofranz ah right - I was looking at the deprecated one - that one must have been through it's deprecation & been removed

Easiest is for the extension to take a copy of the function into the extension, although a better approach would be to replace the custom search (which we are winding down support for) with a search kit display.

Presumably the extension works other than that search & just the search could be removed/ disabled?

**Comment by VangelisP** [comment]
@jofranz while I don't want to overtake this discussion, just wanted to say that I was hitting the same issue and I've spotted on your initial post that you've been using a bash script that you're calling to detect removed code from extension(s): `check-extensions-for-removed-core-code.sh` , what is this and where can I find it :smile: ?
